projects/components/table/src/services/table-settings.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+export interface IPsTableSetting {
+  columnBlacklist: string[];
+  sortColumn: string;
+  sortDirection: 'asc' | 'desc';
+  pageSize: number;
+}
+
+@Injectable({ providedIn: 'root' })
+export class PsTableSettingsService {
+  public defaultPageSize$: Observable<number> = of(15);
+  public settings$: Observable<{ [id: string]: IPsTableSetting }> = of({});
+
+  public settingsEnabled = false;
+  public pageSizeOptions = [5, 10, 25, 50];
+
+  public save(_: string, __: IPsTableSetting): Observable<void> {
+    return of(null);

```

Comment:
null ist eigentlich nicht void

Reply:
Kommt aber nahe ran und of() braucht irgend nen Wert, sonst emitted es nicht. Was wäre dein Vorschlag?

Reply:
Ohne Parameter emitted das of() nicht und null is das näheste was an void ran kommt. Wie wurdest du es denn machen?

Reply:
empty()

Reply:
sonst ist der typ Observable<void> falsch und sollte Observable<null> sein oder? also wenn mans ganz genau nimmt

Reply:
Der Typ is richtig, da die BaseClass und alle Verwendungen Observable void vorschreiben/erwarten. Wenn dann da ein null rein kommt, ises unnütz, aber würde zumindest nixkeputt machen, da ja keiner einen Wert erwartet und somit ignoriert wird. Aber ich machs zu empty, das kannte ich garned. Ich kannte nur of() und NEVER^^